tests/bluetooth/mesh_shell/prj.conf

```diff
@@ -22,15 +22,14 @@ CONFIG_BT_PERIPHERAL=y
 CONFIG_BT_TINYCRYPT_ECC=y
 CONFIG_BT_DEVICE_NAME="Zephyr Mesh"
 CONFIG_BT_GATT_CACHING=n
+CONFIG_BT_MAX_CONN=2
```

Comment:
Why was this changed?

Reply:
Same reason as above.

Reply:
If this is not set to 2, the mesh_shell will give this error when doing "mesh prov pb-gatt on" -> "<err> bt_mesh_adv_ext: adv_start: Advertising failed: err -12"